deploy/roles/elasticsearch-config/tasks/main.yml

```diff
@@ -153,3 +153,23 @@
     name: elasticsearch
     state: started
     enabled: yes
+
+- name: Set recursive admin permissions on log folders
+  acl:
+    path: "{{ item }}"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rX
+    recursive: yes
+    state: present
+  with_items: "{{ es_log_file_locations }}"
+
+- name: Set default admin permissions on log folders
+  acl:
+    path: "{{ item }}"
+    entity: "{{ restricted_group_name }}"
+    etype: group
+    permissions: rX
+    default: yes
+    state: present
+  with_items: "{{ es_log_file_locations }}"
```

Comment:
@stirlingv Since you're already doing Elasticsearch, let's do the Logstash and Kibana as well, which have their own roles, and logs at /var/log/logstash and /var/log/kibana respectively.  Thanks!